### PR TITLE
Added the cypress tests to prow.

### PR DESCRIPTION
**Pull request by lindtdev**
This commit adds a prow job that will execute the cypress tests.

It also adds the ci skript that is executed by the prow job.

### CODE REVIEW

File: .prow.yaml [hunk 1]
```diff
@@ -31,3 +31,20 @@ presubmits:
...
             memory: 512Mi
+
+  - name: pre-docs-run-cypress-tests
+    always_run: true
+    decorate: true
+    optional: true
+    clone_uri: "ssh://git@github.com/kubermatic/docs.git"
+    spec:
+      containers:
+      - image: quay.io/kubermatic/e2e-kind-cypress:1.1.1
```
Comment (xrstf) [inline .prow.yaml:42]: The image is called `quay.io/kubermatic/e2e-kind-cypress:v1.1.1`.

**Comment by lindtdev** [comment]
Re-forking docs broke the PR, I will open a new one.

**Pull request closed without merging**